Sources/SwiftGen/ParserCLI.swift

```diff
@@ -100,6 +100,12 @@ extension ParserCLI {
       description: "generate code for custom yaml configuration files",
       pathDescription: "YAML files (or directories that contain them) to parse."
     ),
+    .init(
+      parserType: Files.Parser.self,
+      name: "files",
+      description: "generate code for referencing specified files",
+      pathDescription: "Files and/or directories to recursively search"
+    ),
```

Comment:
Perfect 👍 
Just a nitpick: we usually keep that list of `.init` ordered by the `name` property, so the parser for your new `"files"` command should go before the one for `"fonts"`, It doesn't change anything in the behavior, but it's just to satisfy our OCD 😅

Reply:
Ah. I guess my OCD didn't look close enough. I saw they weren't exactly in order by `parserType` so I placed it at the end :)

I'll change it up